### DISCUSSION TITLE
How to test different wallets?

### DISCUSSION THREAD

**Discussion by quinn-eschenbach** [wevm/wagmi#1922] [Q&A]
I'm currently trying to integrate new wallets into my dapp. 
However I cant fully disconnect, meaning when i dsiconnect and connect again it automatically connects to metamask and doesnt let me choose which wallet i want to use. 
I tried to remove the page from Metamask and deleted all cookies and the local storage, but it still goes straight to metamask.
Any ideas what i can do?
Thanks in advance for any answers :)

**Comment by SaadNiaziDev** [comment]
disable autoconnect bit in wagmi client.

https://wagmi.sh/react/client#autoconnect-optional

additionally use useDisconnect hook. it will delete any related data stored in session or localstorage
